source/cmake/daemon.cmake

```diff
@@ -0,0 +1,193 @@
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/time_from_rtc.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/qtserialublox.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/daemon.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/pigpiodhandler.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/custom_io_operators.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/filehandler.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/calibration.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/logparameter.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/gpio_mapping.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/tdc7200.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/logengine.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/geohash.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/addresses.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/custom_i2cdetect.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cbusses.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/adafruit_ssd1306/adafruit_ssd1306.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/ads1015/ads1015.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/ads1115/ads1115.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/bme280/bme280.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/bmp180/bmp180.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/eeprom24aa02/eeprom24aa02.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/hmc5883/hmc5883.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/lm75/lm75.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/mcp4728/mcp4728.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/pca9536/pca9536.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/sht21/sht21.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/ubloxi2c/ubloxi2c.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/x9119/x9119.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/Adafruit_GFX.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/i2cdevices/i2cdevice.h"
+    "${MUONDETECTOR_DAEMON_HEADER_DIR}/i2c/linux/i2c-dev.h"
+    )
+
+set(MUONDETECTOR_LOGIN_SOURCE_FILES
+    "${MUONDETECTOR_DAEMON_SRC_DIR}/muondetector-login.cpp"
+    )
+
+set(MUONDETECTOR_LOGIN_INSTALL_FILES
+    "${MUONDETECTOR_DAEMON_CONFIG_DIR}/muondetector-login"
+    )
+
+set(MUONDETECTOR_DAEMON_INSTALL_FILES
+    "${MUONDETECTOR_DAEMON_CONFIG_DIR}/muondetector.conf"
+    )
+
+configure_file(
+    "${MUONDETECTOR_DAEMON_CONFIG_DIR}/muondetector-daemon.1"
+    "${CMAKE_CURRENT_BINARY_DIR}/muondetector-daemon.1"
+    )
+configure_file(
+    "${MUONDETECTOR_DAEMON_CONFIG_DIR}/muondetector-login.1"
+    "${CMAKE_CURRENT_BINARY_DIR}/muondetector-login.1"
+    )
+
+
+add_executable(muondetector-login ${MUONDETECTOR_LOGIN_SOURCE_FILES})
+
+set_target_properties(muondetector-login PROPERTIES POSITION_INDEPENDENT_CODE 1)
+
+target_include_directories(muondetector-login PUBLIC
+    $<BUILD_INTERFACE:${LIBRARY_INCLUDE_DIR}>)
+
+target_link_libraries(muondetector-login
```

Comment:
I have to include pthread to the target_link_libraries directive. Why it is missing now (even since some time in the past)?

Reply:
Fixed with https://github.com/MuonPi/muondetector/commit/2f288cc0a3a4f6b65f65f69a895f3d201b8292ad#diff-41cf6794ba4200b839c53531555f0f3998df4cbb01a4d5cb0b94e3ca5e23947d